### ISSUE TITLE
Can you show more details about the data preparation？ 

### ISSUE THREAD

**Issue by Jugerr** [gazeai/STDN-PyTorch#3]
Like which dataset， and the strategy to split train and test set.
